src/core/trtserver.cc

```diff
@@ -225,6 +228,7 @@ class TrtServerOptions {
  private:
   std::string server_id_;
   std::string repo_path_;
+  bool model_polling_;
```

Comment:
Update constructor to default this to true. This may conflict with the in-flight TRTSERVER metrics change...